learn_script.py

```diff
@@ -4,39 +4,36 @@
 import shlex
 import subprocess
 import sys
-
+import argparse
 
 try:
     from ansimarkup import parse, ansiprint
 except:
-    ansiprint = lambda x: print(re.sub('<.*?>', '', x))
-
-
-script = open(sys.argv[1]).readlines()
+    def ansiprint(x): return print(re.sub('<.*?>', '', x))
 
-help_txt = []
-expect = ''
 
-
-def checkfile(fname, exists=True):
+def checkfile(content, exists=True):
     if exists ^ os.path.exists(content):
-        ansiprint("<fg red> Arquivo/diretório " + ('inexistente' if exists else 'existente') + ": " + content + ". Procure por ajuda.")
+        ansiprint("<fg red> Arquivo/diretório " + ('inexistente' if exists else 'existente') +
+                  ": " + content + ". Procure por ajuda.")
         return False
     return True
 
 
 def prompt():
-    ansiprint("<bold><green>[In]</green></bold> <light-blue>" + os.path.abspath('.') + "</light-blue>$ ", end='')
+    ansiprint("<bold><green>[In]</green></bold> <light-blue>" +
+              os.path.abspath('.') + "</light-blue>$ ", end='')
 
 
 def query_user(content):
-    fancy_input = lambda : prompt() or input()
+    def fancy_input(): return prompt() or input()
```

Comment:
avoid function definition inside other functions and on one-liners.

```suggestion
    fancy_input = lambda : prompt() or input()
```

Reply:
:+1: